scipy/stats/mstats_basic.py

```diff
@@ -356,39 +356,171 @@ def msign(x):
 
 
 def pearsonr(x, y):
-    """
-    Calculates a Pearson correlation coefficient and the p-value for testing
-    non-correlation.
+    r"""
+    Pearson correlation coefficient and p-value for testing non-correlation.
 
-    The Pearson correlation coefficient measures the linear relationship
-    between two datasets. Strictly speaking, Pearson's correlation requires
-    that each dataset be normally distributed. Like other correlation
+    The Pearson correlation coefficient [1]_ measures the linear relationship
+    between two datasets.  The calculation of the p-value relies on the
+    assumption that each dataset is normally distributed.  (See Kowalski [3]_
+    for a discussion of the effects of non-normality of the input on the
+    distribution of the correlation coefficient.)  Like other correlation
     coefficients, this one varies between -1 and +1 with 0 implying no
-    correlation. Correlations of -1 or +1 imply an exact linear
-    relationship. Positive correlations imply that as `x` increases, so does
-    `y`. Negative correlations imply that as `x` increases, `y` decreases.
+    correlation. Correlations of -1 or +1 imply an exact linear relationship.
+    Positive correlations imply that as x increases, so does y. Negative
+    correlations imply that as x increases, y decreases.
```

Comment:
According to [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), the backticks were correct, no?

Reply:
I think I will remove this statement: it implies a deterministic relationship between x and y in my view